blocks/job-openings/job-openings.js

```diff
@@ -0,0 +1,87 @@
+import {
+  decorateIcons,
+} from '../../scripts/lib-franklin.js';
+
+let jobListOffset = 0;
+const selectedRole = 'All categories';
+const selectedLocation = 'All categories';
+
+function addCardsToCardList(cards, cardList) {
+  cards.forEach((card) => {
+    const jobListItem = document.createElement('li');
+    jobListItem.classList.add('job-opening-list-item');
+
+    jobListItem.innerHTML = `
+    <a href="https://www.smartrecruiters.com/Netcentric/${
+  card.id
+}" target="_blank" rel="noopener noreferrer">
+      <span class="position">${card.name}</span>
+      <span class="location">${card.location.country.toUpperCase()} - ${
+  card.location.city
+}</span>
+    </a>`;
+
+    cardList.appendChild(jobListItem);
+  });
+}
+
+function createJobList(parent, cards = []) {
+  const results = document.createElement('p');
+  results.classList.add("job-openings-results");
+  
+  const jobOpeningList = document.createElement('ul');
+  jobOpeningList.classList.add('job-openings-list');
+
+  addCardsToCardList(cards, jobOpeningList);
+
+  parent.appendChild(results);
+  parent.appendChild(jobOpeningList);
+}
+
+function createCTASection(parent, callback) {
+  const buttonRow = document.createElement('div');
+  buttonRow.classList.add('button-row');
+  buttonRow.innerHTML = '<button id="load-more-button" class="button primary">Show More</button>';
```

Comment:
please add TODOs to all static texts, it will be really hard later to find them

Reply:
done